admin-dev/themes/new-theme/scss/pages/product/product_page.scss

```diff
@@ -1149,6 +1149,43 @@ $product-page-padding-bottom: 80px !default;
     }
   }
 
+  #product_description_packed_products {
```

Comment:
To avoid relying on the ID (since the Packs are moved into another tab and the ID changes completely) I think it would be safer to introduce a CSS class on the initial container that we can use regardless of the initial ID (just in case the design changes again and the Packs are moved somewhere else ^^)